pkg/controller/handlers/toolreference/migrate.go

```diff
@@ -0,0 +1,57 @@
+package toolreference
+
+import (
+	"github.com/obot-platform/nah/pkg/router"
+	v1 "github.com/obot-platform/obot/pkg/storage/apis/obot.obot.ai/v1"
+)
+
+var toolMigrations = map[string]string{
+	"file-summarizer-file-summarizer": "file-summarizer",
+}
+
+func (h *Handler) MigrateToolNames(req router.Request, _ router.Response) error {
```

Comment:
Instead of this being a controller, how do you feel about it being in the controller `PreStart` [here](https://github.com/obot-platform/obot/blob/main/pkg/controller/controller.go#L39)? That way, it only runs on startup.

Reply:
I feel like it is straightforward to do it in a handler. Prestart feel like it is mostly initializing data, also not sure if it is utilizing cache so it might slow things down.

Reply:
Doing it as a pre-start won't slow things down too much because the new container isn't healthy until the controllers start up. If that is a concern, then you could always add it to the post-start.

I don't really like doing this as a handler because the migration really only needs to run once and not every time there are changes. Additionally, there is long-term work that leads to not handling objects on startup. This won't work anymore when that change goes in.

Reply:
ok, I can try to move it.